internal/services/subscription/subscription_resource.go

```diff
@@ -118,6 +118,24 @@ func resourceSubscription() *pluginsdk.Resource {
 				Computed:    true,
 			},
 
+			"subscription_owner_id": {
+				Type:         pluginsdk.TypeString,
+				Description:  "Owner ID of the subscription",
+				Optional:     true,
+				ForceNew:     true,
+				RequiredWith: []string{"subscription_tenant_id"},
+				ValidateFunc: validation.IsUUID,
+			},
+
+			"subscription_tenant_id": {
```

Comment:
To the best of my knowledge, this property can change the tenant membership of the subscription, is this necessary to set the ownerID? Do you have example output of what happens when this value is not supplied with `SubscriptionOwnerId` ?

Reply:
This is the received error when creating a subscription and supplying owner ID without tenant ID:
```
│ Error: creating new Subscription (Alias "XXX-XXX-XXX-XXX"): performing AliasCreate: subscriptions.SubscriptionsClient#AliasCreate: Failure sending request: StatusCode=0 -- Original Error: Code="InvalidInput" Message="service owner OID cannot be specified without a service tenant ID specified"
```